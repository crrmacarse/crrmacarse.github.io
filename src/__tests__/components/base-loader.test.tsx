import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { create, ReactTestRenderer } from 'react-test-renderer';
import BaseLoader from 'components/shared/loader/base';

let wrapper: ShallowWrapper;
let snapshot: ReactTestRenderer;

beforeEach(() => {
  wrapper = shallow(<BaseLoader />)
  snapshot = create(<BaseLoader />);
});

describe('<BaseLoader />', () => {
  it('matches the snapshot', () => {
    expect(snapshot.toJSON()).toMatchSnapshot();
  });

  it('renders', () => { wrapper })
});
