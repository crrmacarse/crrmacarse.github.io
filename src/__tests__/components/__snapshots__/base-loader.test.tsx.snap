// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseLoader /> matches the snapshot 1`] = `
<div
  className="base-loader"
>
  <div
    className="overlay fixed w-full h-screen top-0 z-19"
  />
  <div
    className="fixed flex flex-col justify-center items-center w-full h-screen top-0 z-20"
  />
</div>
`;
